api/src/pcapi/routes/backoffice/offerers.py

```diff
@@ -0,0 +1,188 @@
+import sqlalchemy as sa
+
+from pcapi.core.bookings import models as bookings_models
+from pcapi.core.educational import models as educational_models
+from pcapi.core.finance import models as finance_models
+from pcapi.core.offerers import models as offerers_models
+from pcapi.core.offers import models as offers_models
+from pcapi.core.permissions import models as perm_models
+from pcapi.core.permissions import utils as perm_utils
+from pcapi.models import db
+from pcapi.models.api_errors import ApiErrors
+from pcapi.serialization.decorator import spectree_serialize
+
+from . import blueprint
+from . import serialization
+from . import utils
+
+
+@blueprint.backoffice_blueprint.route("offerers/<int:offerer_id>", methods=["GET"])
+@spectree_serialize(
+    response_model=serialization.Response,
+    on_success_status=200,
+    api=blueprint.api,
+)
+@perm_utils.permission_required(perm_models.Permissions.READ_OFFERER)
+def get_offerer(offerer_id: int) -> serialization.Response:
+    ko_bank_informations_query = (
+        sa.select(sa.func.count(offerers_models.Venue.id))
+        .select_from(offerers_models.Venue)
+        .outerjoin(finance_models.BankInformation)
+        .filter(
+            offerers_models.Venue.managingOffererId == offerers_models.Offerer.id,
+            sa.or_(
+                finance_models.BankInformation.status.is_(None),
+                finance_models.BankInformation.status != finance_models.BankInformationStatus.ACCEPTED,
+            ),
+        )
+    )
+    ok_bank_informations_query = (
+        sa.select(sa.func.count(offerers_models.Venue.id))
+        .select_from(offerers_models.Venue)
+        .outerjoin(finance_models.BankInformation)
+        .filter(
+            offerers_models.Venue.managingOffererId == offerers_models.Offerer.id,
+            finance_models.BankInformation.status == finance_models.BankInformationStatus.ACCEPTED,
+        )
+    )
+    active_individual_offers_query = (
+        sa.select(sa.func.count(offers_models.Offer.id))
+        .select_from(offerers_models.Venue)
+        .join(offers_models.Offer)
+        .filter(
+            offerers_models.Venue.managingOffererId == offerers_models.Offerer.id,
```

Comment:
On utilise `Offerer.id`, mais je ne vois pas de jointure. Est-ce qu'on ne veut pas juste ça : 

```suggestion
            offerers_models.Venue.managingOffererId == offerer_id,
```

Idem dans d'autres requêtes plus bas.

Reply:
Je pense que cette requête va être très longue pour certaines structures qui ont beaucoup d'offres. Je viens de tester sur staging avec l'offerer_id = 1869. J'ai coupé au bout d'une minute... Idem pour la requête qui renvoie le nombre d'offres individuelles inactives.

Reply:
je suis d'accord